build.gradle

```
@@ -1,5 +1,5 @@
 allprojects {
-    pluginManager.withPlugin('maven-publish') {
+    pluginManager.withPlugin('org.jreleaser:1.18.0') {
```

```suggestion
    pluginManager.withPlugin('org.jreleaser') {
```

This should just be the name of the plugin

---

When the version is not specified, the build fails with following error:

> An exception occurred applying plugin request [id: 'org.jreleaser', version: '1.18.0']
> Failed to apply plugin 'org.jreleaser'.
> &nbsp;&nbsp;&gt; A problem occurred evaluating script.
> &nbsp;&nbsp;&nbsp;&nbsp;&gt; Could not find method publishing() for arguments [publish_5th5obhuqf1kwj8qwjaan0fet$_run_closure1@2acda146] on project ':contentgrid-hateoas-spring' of type org.gradle.api.Project.